Wikipedia/Code/ArticleViewController.swift

```diff
@@ -1045,6 +1045,14 @@ private extension ArticleViewController {
         if let altTextExperimentViewModel {
             self.navigationItem.titleView = nil
             self.title = altTextExperimentViewModel.localizedStrings.articleNavigationBarTitle
+
+            let rightBarButtonItem = UIBarButtonItem(
+                image: UIImage(systemName: "ellipsis.circle"),
+                primaryAction: nil,
+                menu: overflowMenu)
+            navigationItem.rightBarButtonItem = rightBarButtonItem
+            rightBarButtonItem.tintColor = .link
```

Comment:
Our theming is a little weird (we can talk more about it!), so we get colors from the theme like this:

`theme.colors.link`

This class already references the Theme class that propagates theme changes across the app.